components/TripPreview.js

```diff
@@ -1,28 +1,31 @@
 import styled from "styled-components";
 import Image from "next/image";
+import Link from "next/link";
 
 export default function TripPreview({ trip }) {
   return (
     <StyledArticle>
-      <StyledImageWrapper>
-        <StyledImage
-          src={trip.image}
-          height={800}
-          width={800}
-          alt={trip.location}
-        />
-      </StyledImageWrapper>
-      {trip.location}
-      <br />
-      {`${trip.startDate.substring(8)}/${trip.startDate.substring(
-        5,
-        7
-      )}/${trip.startDate.substring(2, 4)}`}{" "}
-      -{" "}
-      {`${trip.endDate.substring(8)}/${trip.endDate.substring(
-        5,
-        7
-      )}/${trip.endDate.substring(2, 4)}`}
+      <StyledLink href="/explore">
+        <StyledImageWrapper>
+          <StyledImage
+            src={trip.image}
+            height={800}
+            width={800}
+            alt={trip.location}
+          />
+        </StyledImageWrapper>
+        {trip.location}
+        <br />
+        {`${trip.startDate.substring(8)}/${trip.startDate.substring(
```

Comment:
i don't understand that codeline, could you explain what substring does?

Reply:
Substring is a string method that takes only a part of a string. You add one value which is the starting value and you can add a second one which is the end of the substring, for example neuefische.substring(4, 9) would give you a new string which starts at the letter with index 4 (f) and ends before the letter with index 9 (e), so you would effectively end up with the string "fisch." I used it to format the date. In my array each date is in this format yyyy-mm-dd but I want it to be displayed as dd/mm/yy.